### ISSUE TITLE
Debian build is missing dependencies / imports

### ISSUE THREAD

**Issue by nbuchwitz** [jpmens/msoak#4]
With Debian Buster at least the following has be installed:

``libconfig-dev`` (already stated)
``liblua5.3-dev`` (not 5.2 as mentioned in README)
``libmosquitto-dev`` (missing)
``libbsd-dev`` (missing?)

There are also some changed necessary. Otherwise the build process will complain about implicit declaration of strlcpy. I guess the include of ``bsd/string.h`` is only needed on Linux, so a conditional import would make sense. 
```
diff --git a/Makefile b/Makefile
index 8e370d4..a12d4b9 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CFLAGS=-Wall -Werror -I. `pkg-config --cflags lua5.3`
-LDFLAGS=-L /usr/local/lib -lmosquitto -lconfig `pkg-config --libs lua5.3`
+LDFLAGS=-L /usr/local/lib -lmosquitto -lconfig `pkg-config --libs lua5.3` `pkg-config --libs libbsd`
 
 BINDIR = /usr/local/bin
 MANDIR  = /usr/local/share/man
diff --git a/msoak.c b/msoak.c
index 5a5e332..b023f36 100644
--- a/msoak.c
+++ b/msoak.c
@@ -20,6 +20,7 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <string.h>
+#include <bsd/string.h>
 #include <unistd.h>
 #include <errno.h>
 #include <signal.h>
```

If I'm not missing something and you agree, I'm happy to send a PR.

**Comment by jpmens** [comment]
I'll gladly take the PR (note: Makefile has just changed a bit)

**Comment by nbuchwitz** [comment]
Should be fixed by #7

**Comment by jpmens** [comment]
closed via #7
